.github/workflows/build-nflow-explorer-ng.yaml

```diff
@@ -0,0 +1,35 @@
+name: Build and test nflow-explorer-ng
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+    paths-ignore:
+      - '**/*.md'
+concurrency:
+  group: build-nflow-explorer-ng-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+permissions:
+  contents: read
+  checks: write
+  security-events: write
+jobs:
+  build-and-test-nflow-explorer-ng:
```

Comment:
I would put this job to the build.yaml with other jobs until duplicating the boilerplate around the job can be avoided.

Reply:
Doing it in separate .yaml makes it possible to run it in parallel with main build, thus making build faster.

Reply:
Different jobs in same file are ran in parallel, so having it in a separate file does not make build any faster.

Reply:
fixed